v22.2/backup.md

```diff
@@ -45,14 +45,20 @@ To view the contents of an backup created with the `BACKUP` statement, use [`SHO
 
 ## Required privileges
 
+{% include {{ page.version.version }}/backups/updated-backup-privileges.md %}
+
+### Existing required privileges
```

Comment:
Hmm, this is where it'd be great to have a name of the new priv model from marketing / product. "Required privileges" is actually "now and future required privs" while "Existing required privileges" is actually "now and past required privs". This is very tricky!

APIs get to go `v2`, `v3`. CRDB gets to go `v22.1`, `v22.2`. But this feature v2.0 doesn't have a name other than (`new` && `delivered with CRDB v22.2`), so we have to get clever ourselves:

Here's some brainstorming:

> ## Legacy privileges - CockroachDB v22.2 and previous
> ## Granular privileges - CockroachDB v22.2 and later

Or maybe

> ## Required privileges using the legacy privilege model
>
> The following privileges are required to use `BACKUP` when using the legacy privilege model used in CockroachDB up to and including v22.2. Support for the legacy privilege model will be removed in a future release:
> ...
> ## Required privileges using the granular privilege model
> The following privileges are required to use `BACKUP` when using the granular privilege model introduced in CockroachDB v22.2. The granular privilege model will be the sole supported privilege model (replacing the [legacy privilege model]) in a future release:
> ...

Just brainstorming!! This is tricky and I'm not sure how to solve.

Reply:
So a per Andrew and mine's discussion on Slack, we opted for the new, fine-grained privilege model, being simply ## Required Privileges. Since this is what we now recommend, let's not draw a distinction between the two. We want this model to be our main and only privilege model in the future. 

"Legacy" is exactly the word we need for the existing model and provides the distinction we need. Going to mark this up and add it as another H2 header, so it doesn't "fall under" the main Required Privileges.